splits/README.md

```diff
@@ -27,6 +27,10 @@ Current splits ([semaphore legend](../README.md#split-semaphore)):
   - 🟢 `human_hard`
   - 🟢 `balanced`
   - 🟢 `mixed_vs_human_2`
+- Secondary Structure
+  - 🟠 `sampled`
```

Comment:
Even tho these two sets are sampled, they are sampled from redundancy reduced sequence sets, so they are "safe" to be sampled ;)

I would turn these green :) It's ok!

Reply:
Changed.